mkdocs.yml

```diff
@@ -15,83 +15,83 @@ edit_uri: edit/main/docs/
 
 # Plugins
 plugins:
-   - include-markdown
-   - tags
-   - search
```

Comment:
Please avoid making indentation changes throughout the file, as it is managed by the Documentation team. Only update the lines where changes are required.